pkg/build/builder/dockerutil.go

```diff
@@ -156,15 +156,27 @@ func dockerRun(client DockerClient, createOpts docker.CreateContainerOptions, lo
 
 	containerName := containerNameOrID(c)
 
-	// Container was created, so we defer its removal.
-	defer func() {
+	// Container was created, so we defer its removal, and also remove it if we get a sigterm
```

Comment:
This line got a bit buried in the discussion above. Would be nice to get it to what we have in the source-to-image PR:
```
Container was created, so we defer its removal, and also remove it if we get a SIGINT/SIGTERM/SIGQUIT.
```